extensions/2.0/Vendor/EXT_feature_metadata/0.0.0/README.md

```diff
@@ -0,0 +1,682 @@
+
+Feature textures are accessed directly by texture coordinates, rather than feature IDs. This makes them well-suited for representing properties that vary with position such as elevation heightmaps or vector fields.
+
+Feature textures comply with the [Metadata Texture Encoding](https://github.com/CesiumGS/3d-tiles/blob/3d-tiles-next/specification/Metadata/0.0.0/README.md#metadata-texture-encoding) of the Cesium 3D Metadata Specification. However, the terminology in this extension is slightly different; "feature texture" is used instead of the more generic "metadata texture".
+
+**Note**: in the core glTF standard, only PNG and JPEG textures are supported. This limits data types to those based on `UINT8`:
+
+* `UINT8`
+* normalized `UINT8` (the texture stores [0, 255], but values are remapped to [0, 1])
+* `ARRAY` of `UINT8` with `componentCount` of `1-4`
+* `ARRAY` of normalized `UINT8` with `componentCount` of `1-4`
+
+In this glTF extension, textures are defined with the following steps:
+1. A class is defined in the root `EXT_feature_metadata.classes` object. This is used to describe the metadata in the texture.
+2. A feature texture is defined in the root `EXT_feature_metadata.featureTextures` object. This must reference the class ID defined in step 1.
+3. A feature texture is associated with a primitive by listing the feature texture ID in the `primitive.EXT_feature_metadata.featureTextures` array.
+
+Example:
+
+```json
+{
+  "extensions": {
+    "EXT_feature_metadata": {
+      "classes": {
+        "elevation": {
+          "name": "Elevation",
+          "properties": {
+            "elevation": {
+              "name": "Elevation (m)",
+              "description": "Elevation above the WGS84 ellipsoid"
+            }
+          }
+        }
+      },
+      "featureTextures": {
+        "elevationTexture": {
+          "class": "elevation",
+          "properties": {
+            "elevation": {
+              "texture": {
+                "index": 0,
+                "texCoord": 0
+              },
+              "channels": "r"
+            }
+          }
+        }
+      }
+    }
+  },
+  "images": [
+    {
+      "mimeType": "image/png",
+      "name": "Elevation Heightmap",
+      "uri": "elevation.png"
+    }
+  ],
+  "textures": [
+    {
+      "name": "Elevation Texture",
+      "source": 0
+    }
+  ],
+  "meshes": [
+    {
+      "name": "Feature Texture Example",
+      "primitives": [
+        {
+          "attributes": {
+            "POSITION": 0,
+            "NORMAL": 1,
+            "TEXCOORD_0": 2
+          },
+          "indices": 3,
+          "material": 0,
+          "extensions": {
+            "EXT_feature_metadata": {
+              "featureTextures": ["elevationTexture"]
+            }
+          }
+        }
+      ]
+    }
+  ],
+}
+```
+
+## Feature Identification
```

Comment:
Today I realized an omission in this spec: for per-face metadata, we are just using vertex metadata where every vertex in the face must have the same feature ID

This should be described in one of these sub-sections (or maybe make a new one?)